src/App.tsx

```diff
@@ -34,19 +34,25 @@ import { appStart, setArtboards, setSelectedArtboard, updateActiveArtboardLayers
 import { redo, undo } from './modules/history/actions';
 import store from './store';
 import { RootState } from './store/rootReducer';
-import { Artboard, colorSpaceType } from './types';
+import { Artboard, colorSpaceType, guidesRefType, snappingObjectType } from './types';
 import { generateId } from './utils';
 import { SmartObject } from './modules/reflection/helpers';
 import { useModalStyles } from './styles/modal';
 import SectionTitle from './components/SectionTitle';
+import { FABRIC_JSON_ALLOWED_KEYS } from './constants';
+import { createSnappingLines, filterSnappingLines, snapToObject } from './modules/snapping';
 
 store.dispatch(appStart());
 
 function App() {
 	const dispatch = useDispatch();
 	const artboards = useSelector((state: RootState) => state.app.artboards);
 	const selectedArtboard = useSelector((state: RootState) => state.app.selectedArtboard);
-
+	const [snapDistance] = useLocalStorage<string>({
```

Comment:
@akram-r we can keep this in local only, don't need to set in localstorage for now